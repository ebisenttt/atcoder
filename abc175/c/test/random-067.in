636881730361944 678128092905816 69295796388186
