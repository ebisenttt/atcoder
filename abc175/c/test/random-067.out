56076233519916
